prql-php/src/Compiler.php

```diff
@@ -9,179 +9,261 @@
  * PHP version 8.0
  *
  * @api
- * @package   Prql\Compiler
+ *
  * @author    PRQL
  * @copyright 2023 PRQL
  * @license   https://spdx.org/licenses/Apache-2.0.html Apache License 2.0
- * @link      https://prql-lang.org/
+ *
+ * @see      https://prql-lang.org/
  */
 
 declare(strict_types=1);
 
 namespace Prql\Compiler;
 
 /**
- * The PRQL compiler transpiles PRQL queries.
- *
- * @package Prql\Compiler
  * @author  PRQL
  * @license https://spdx.org/licenses/Apache-2.0.html Apache License 2.0
- * @link    https://prql-lang.org/
+ *
+ * @see    https://prql-lang.org/
  */
 final class Compiler
```

Comment:
I've been thinking about if the `Compiler` class should be named `PrqlCompiler` instead... anyone have any thoughts?